sympy/functions/elementary/integers.py

```diff
@@ -150,25 +157,32 @@ class ceiling(RoundFunction):
     value not less than its argument. Ceiling function is generalized
     in this implementation to complex numbers.
 
-    More information can be found in "Concrete mathematics" by Graham,
-    pp. 87 or visit http://mathworld.wolfram.com/CeilingFunction.html.
-
-        >>> from sympy import ceiling, E, I, Float, Rational
-        >>> ceiling(17)
-        17
-        >>> ceiling(Rational(23, 10))
-        3
-        >>> ceiling(2*E)
-        6
-        >>> ceiling(-Float(0.567))
-        0
-        >>> ceiling(I/2)
-        I
+    Examples
+    ========
+
+    >>> from sympy import ceiling, E, I, Float, Rational
+    >>> ceiling(17)
+    17
+    >>> ceiling(Rational(23, 10))
+    3
+    >>> ceiling(2*E)
+    6
+    >>> ceiling(-Float(0.567))
+    0
+    >>> ceiling(I/2)
+    I
 
     See Also
     ========
 
-    floor
+    sympy.functions.elementary.integers.floor
```

Comment:
For these top level functions I'm not sure it helps to point to the actual location. Seems to confuse more than help the end user.